docs/source/tests.rst

```diff
@@ -35,3 +35,119 @@ If you are adding tests or altering functionality, it might be faster to only ru
 .. code-block:: sh
 
   python -m unittest neo/test_settings.py
+
+  
+
+Guideline
```

Comment:
Please change the title to: `Fixtures Guidelines`
to make it more explicit what the next section is about.